kpatch-build/create-diff-object.c

```diff
@@ -3234,6 +3313,13 @@ static void kpatch_create_intermediate_sections(struct kpatch_elf *kelf,
 				special = true;
 
 		list_for_each_entry_safe(rela, safe, &sec->relas, list) {
+			/*
+			 * s390_indirect_jump is a global hidden symbol and is
+			 * present in each object file which references it.
+			 * They can be used as normal relas.
+			 */
+			if (strstr(rela->sym->name, "s390_indirect_jump"))
+				continue;
```

Comment:
Are there instances where `s390_indirect_jump` relocation would have `rela->need_dynrela` set?  FWIW, when building the integration tests, I didn't see any examples where this happened.

Reply:
Yes. You are right. I removed it.